bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/component.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
+import Button from '/imports/ui/components/button/component';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import { styles } from './styles';
+
+const intlMessages = defineMessages({
+  downloadPresentationButton: {
+    id: 'app.downloadPresentationButton.label',
+    description: 'Download presentation label',
+  },
+});
+
+const propTypes = {
+  intl: intlShape.isRequired,
+  handleDownloadPresentation: PropTypes.func.isRequired,
+  dark: PropTypes.bool,
+  elementName: PropTypes.string,
+};
+
+const defaultProps = {
+  dark: false,
+  elementName: '',
+};
+
+const DownloadPresentationButton = ({
+  intl, handleDownloadPresentation, dark, elementName,
+}) => {
+  const formattedLabel = intl.formatMessage(
+    intlMessages.downloadPresentationButton,
+    ({ 0: elementName ? elementName.toLowerCase() : '' }),
```

Comment:
This isn't being used by the formatted message.

Reply:
removed, tnx!